test/zdtm.py

```diff
@@ -28,7 +28,15 @@
 
 import yaml
 
-os.chdir(os.path.dirname(os.path.abspath(__file__)))
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+os.chdir(SCRIPT_DIR)
+
+# Instead of passing the location of the image streamer binary as an argument
+# (e.g., --criu-image-streamer-bin), the caller can modify PATH to include the
+# binary location. We add a default location as well.
+IMG_STREAMER_PATH = os.path.realpath(os.path.join(SCRIPT_DIR, "../../criu-image-streamer"))
```

Comment:
can we specify a path to criu-image-streamer as an argument? For example, it can be --image-streamer /x/y/criu-image-streamer

Reply:
We can. Would it be acceptable to just prepend the specified directory in PATH?

Reply:
Sorry, I was out for a while.

> Would it be acceptable to just prepend the specified directory in PATH?

Yes, it would.